megazords/reference-browser/android/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,2 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="mozilla.appservices.composites.reference_browser" />
+    package="mozilla.appservices.reference_browser_megazord" />
```

Comment:
Nit: would `mozilla.appservices.megazords.reference_browser` be better?

Reply:
I don't really mind.  I decided to not have the sub namespace (`mozilla.appservices.megazord...`) simply because it might complicate publishing to `maven.mozilla.org` in #252.

Reply:
I'm going to keep this as is for simplicity of this patch series; we can bump it in the future if there's an advantage to doing so.